docs/dev/glossary.md

```diff
@@ -619,7 +619,7 @@ See also: _[sauce connect proxy](#sauce-connect-proxy)._
 
 ### Tunnel Identifier
 
-The Sauce Connect Proxy test configuration option that allows you to assign a name to your tunnel(s), giving you more control and monitoring capability over the tunnel. If you launch a tunnel without naming it, your test traffic will default to running through that unnamed tunnel. More information: [Using Sauce Connect Tunnel Identifiers](/secure-connections/sauce-connect/setup-configuration/basic-setup#using-tunnel-identifiers).
+The Sauce Connect Proxy test configuration option that allows you to assign a name to your tunnel(s), giving you more control and monitoring capability over the tunnel. If you launch a tunnel without naming it, your test traffic will default to running through that unnamed tunnel. More information: [Using Sauce Connect Tunnel Names](/secure-connections/sauce-connect/setup-configuration/basic-setup#using-tunnel-names).
```

Comment:
@waggledans We should switch the descriptions for Tunnel Identifier and Tunnel Name.
```suggestion
See: [Tunnel Name](#tunnel-name)
```